### ISSUE TITLE
Ability to disable OpenShift authentication

### ISSUE THREAD

**Issue by leoluk** [leoluk/openshift-netbox#5]
Do we always want to authenticate against the OpenShift cluster, or make it configurable?

**Comment by leoluk** [comment]
@toreanderson Do you need this? Low priority otherwise

**Comment by toreanderson** [comment]
No, I thought the OpenShift authentication was an awesome addition! :+1:

Saves me figuring out setting up LDAP auth to our user directory (since the OpenShift rig already has that in place).
